include/z64actor.h

```diff
@@ -454,14 +468,14 @@ typedef enum {
     /* 3 */ DOORLOCK_MAX
 } DoorLockType;
 
-// Targetability / ACTOR_FLAG_TARGETABLE?
-#define ACTOR_FLAG_1             (1 << 0)
-// 
+// Allows Tatl to fly over the actor and Z-target it
+#define ACTOR_FLAG_TARGETABLE    (1 << 0)
```

Comment:
Is Z-target in this context the same as lockOn, or is it the more general case of targeting? It's possible for tatl to fly to an actor because of this flag, but not be able to lockOn which is controlled/blocked by a different actor flag: `ACTOR_FLAG_8000000` or `ACTOR_FLAG_CANT_LOCK_ON`

Reply:
Exactly. As you said having this flag enables Tatl to fly to an actor and lock-on it. To only allow flying over the actor but not lock-on then both flags must be set

Reply:
I was referring more to the comment in that `ACTOR_FLAG_TARGETABLE` alone isn't enough to lockOn. `ACTOR_FLAG_8000000` must also be unset for you to be able to lock on

Reply:
I rewrote this comment a bit, tell me if it is clear now